src/usePagenation/createPagenation.ts

```diff
@@ -0,0 +1,85 @@
+
+
+interface pagenation {
+    current: Number;
+    pageSize: Number;
+    total: Number;
+    data: Array<any> | null;
+    onChange: Function;
+    run: Function;
+}
+
+export default class CreatePagenation {
+    current: Number;
+    pageSize: Number;
+    total: Number;
```

Comment:
`Number` is the constructor, you should use `number` instead.
```suggestion
    current: number;
    pageSize: number;
    total: number;
```